metadata_backend/api/handlers/object.py

```diff
@@ -101,6 +101,13 @@ async def post_object(self, req: Request) -> Response:
         """
         _allowed_csv = ["sample"]
         schema_type = req.match_info["schema"]
+
+        folder_id = req.query.get("folder", "")
```

Comment:
Not sure we do the check if the folder ID is owned by a project the user has access to, otherwise we risk that we get a folder ID from a project the user does not have access to, or do you think this case is plausible ?

probably in #354 we should have checked if it is owned by a user, but we can leave that be and correct it moving forward.

Reply:
True, the project is checked in all other methods (GET, PUT, PATCH, DELETE) but not in POST, I'll add it